### PR TITLE
#8 change styling and small html optimization

### LINKED ISSUE #8: Match app style to the common timetable

### PR DESCRIPTION
**Pull request by AndrzejLech**

### CODE REVIEW

File: src/app/components/common-timetable/timetable/timetable.component.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+p {
+  margin: 0;
+  text-align: center;
+}
+p {
+  margin: 0;
+  text-align: center;
+}
```
Comment (PythSomeone) [inline src/app/components/common-timetable/timetable/timetable.component.scss]: duplication

File: src/app/components/plan-week/plan-week.component.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+p {
+  margin: 0;
+  text-align: center;
+}
+p {
+  margin: 0;
+  text-align: center;
+}
```
Comment (PythSomeone) [inline src/app/components/plan-week/plan-week.component.scss]: duplication

**Review by PythSomeone — CHANGES_REQUESTED** [review]
remove duplicated styles

File: src/app/components/common-timetable/timetable/timetable.component.scss (hunk 1 above)
Comment (AndrzejLech) [reply src/app/components/common-timetable/timetable/timetable.component.scss]: done

File: src/app/components/plan-week/plan-week.component.scss (hunk 2 above)
Comment (AndrzejLech) [reply src/app/components/plan-week/plan-week.component.scss]: done

**Review by PythSomeone — APPROVED** [review]

**Pull request merged**